PittAPI/laundry.py

```diff
@@ -88,34 +88,34 @@ def get_status_detailed(building_name: str) -> List[Dict[str,str]]:
     # Get the weird laundry data
     headers = {"Cookie": "PHPSESSID={}".format(cookie)}
     response = requests.get("http://www.laundryview.com/dynamicRoomData.php?location={}".format(
-        location_dict[building_name]), headers=headers).text
-    resp_split = response.split('&')[3:]
+        location_dict[building_name]), headers=headers)
+    resp_split = response.text.split('&')[3:]
 
-    cleaned_resp = []
+    cleaned_resp = []  # type: List[List[str]]
     for status_string in resp_split:
         if "machine" not in status_string:
             continue
         machine_name = status_string[:status_string.index('=')].replace('Status', '') #type: str
         status_string = status_string[status_string.index('=') + 1:].strip()
 
-        old_machine_split = status_string.split("\n") #type: List[str]
-        old_machine_split[0] += machine_name
+        status_lines = status_string.split("\n") #type: List[str]
+        status_lines[0] += machine_name
```

Comment:
This is another instance where I normally wouldn't have renamed this variable, but my changes conflicted with someone else who renamed it, and I think the name `status_lines` is clearer.